src/spglib.h

```diff
@@ -235,17 +257,13 @@ SpglibDataset *spgat_get_dataset_with_hall_number(
 void spg_free_dataset(SpglibDataset *dataset);
 void spg_free_magnetic_dataset(SpglibMagneticDataset *dataset);
 
-/* Find symmetry operations. The operations are stored in */
-/* ``rotation`` and ``translation``. The number of operations is */
-/* return as the return value. Rotations and translations are */
-/* given in fractional coordinates, and ``rotation[i]`` and */
-/* ``translation[i]`` with same index give a symmetry operations, */
-/* i.e., these have to be used together. */
+SPG_DEPRECATED("Use the variables from SpglibDataset")
 int spg_get_symmetry(int rotation[][3][3], double translation[][3],
```

Comment:
I don't remember `spg_get_symmetry` is deprecated. 
@atztogo Is it?

Reply:
If I remember correctly, this is not deprecated.